content/docs/command-reference/init.md

```diff
@@ -14,106 +14,103 @@ DVC works best in a Git repository. This enables all features, providing the
+> versa. This is because each one has their own Git root.
+
 ### Initializing DVC in subdirectories
 
 `--subdir` must be provided to initialize DVC in a subdirectory of a Git
 repository. DVC still expects to find the Git repository (will check all
-directories up to the root to find `.git`). This options does not affect any
-config files, `.dvc` directory is created the same way as in the default mode.
-This way multiple DVC projects (including nested ones) could be initialized in a
-single Git repository providing isolation and granular project management.
+directories up to the system root to find `.git/`). This options does not affect
+any config files, `.dvc/` directory is created the same way as in the default
+mode. This way multiple <abbr>DVC projects</abbr> can be initialized in a single
+Git repository, providing isolation between projects.
 
 #### When is this useful?
 
-`--subdir` is mostly used in the scenario of a
-[monorepo](https://en.wikipedia.org/wiki/Monorepo), but also can be used in
-other workflows when such isolation and/or advanced granularity is needed.
+This option is mostly useful in the scenario of a
+[monorepo](https://en.wikipedia.org/wiki/Monorepo) (Git repository split into
+several project directories), but can also be used with other patterns when such
+isolation is needed. `dvc init --subdir` mitigates the issues of initializing
+DVC in the Git repo root:
 
-Let's imagine we have an existing Git repository that is split into sub-projects
-(monorepo). In this case `dvc init --subdir` can be run in one or many
-sub-projects to mitigate the issues of initializing in the Git repository root:
+- Repository maintainers might not allow a top level `.dvc/` directory,
+  especially if DVC is being used by several sub-projects (monorepo).
 
-- Repository maintainers might not allow extra `.dvc` top level directory,
-  especially if DVC is being used by a small number of sub-projects.
+- DVC [internals](/doc/user-guide/dvc-files-and-directories) (config file, cache
+  directory, etc.) are shared across different sub-projects. This forces all of
+  them to use the same DVC settings and
+  [remote storage](/doc/command-reference/remote).
 
-- Not enough isolation/granularity - DVC config, cache, and other files are
-  shared across different sub-projects. Means that it's not easy to use
-  different remote storages, for example, for different sub-projects, etc.
-
-- Not enough isolation/granularity - commands like `dvc pull`, `dvc checkout`,
-  and others analyze the whole repository to look for `dvc.yaml` or `.dvc` files
-  to download files and directories, to reproduce <abbr>pipelines</abbr>, etc.
-  It can be expensive in the large repositories with a lot of projects.
-
-- Not enough isolation/granularity - commands like `dvc metrics diff`, `dvc dag`
-  and others by default dump all the metrics, all the pipelines, etc.
+- By default, DVC commands like `dvc checkout` and `dvc repro` explore the whole
```

Comment:
still, case with not isolated/granular `dvc metrics diff` seems a bit different than `dvc pull` - one is mostly about performance, second is about actual output - might resonate with different people.

It's fine though to keep it this way. Not a strong one.

Reply:
Agree. I changed the last sentence to "This can produce undesirable results and/or be inefficient for large monorepos." to better capture both cases.

Reply:
yep, but this is a good example of what I meant with implicit vs explicit. `undesirable results` sounds and reads cool, and one paragraph looks better for sure, but it almost does not mean anything to a reader .. it's hard (or at least takes an effort) to get an example of what does it mean.

again, not a big deal in this case, just want to further explain my thought-process on being explicit if possible.

Reply:
I see. OK, split into 2 bullets for clarity in be5bc6dc.